blocks/fragment/fragment.js

```diff
@@ -38,7 +38,16 @@ export default async function decorate(block) {
   if (fragment) {
     const fragmentSection = fragment.querySelector(':scope .section');
     if (fragmentSection) {
-      block.closest('.section').classList.add(...fragmentSection.classList);
+      const mainSection = block.closest('.section');
+      if (mainSection) {
+        mainSection.classList.add(...fragmentSection.classList);
+        const dataset = { ...mainSection.dataset, ...fragmentSection.dataset };
+        // block dataset takes precedence so that
+        // data-sectionstatus changes to "loaded" (from "initialized")
+        Object.keys(dataset).forEach((key) => {
+          mainSection.setAttribute(`data-${key}`, dataset[key]);
+        });
+      }
```

Comment:
This seems to not work as expected.
<img width="796" alt="Screenshot 2023-11-08 at 22 00 14" src="https://github.com/hlxsites/servicenow/assets/3651689/c63e4407-3f7e-4490-80d7-12eccdc08210">

There are both `data-section-status` and `data-sectionstatus`. 
Same for the extra value: in the fragment it is `data-extra-value` and in the resulting section after loading it is `data-extravalue`.

I believe the main section data set should take precedence over the fragment, as the destination word document configs should be more specific/are overriding what is in the fragment which is more generic. 
Another aspect is: if a section there are multiple blocks and a fragment, the status of the section should not change until all blocks have finished loading, so `data-section-status` should retain the same value after loading the fragment.

Reply:
Fixed in https://github.com/hlxsites/servicenow/pull/69/commits/cdd585c44a7ef81210f86b67fc7b909dcb773827 and https://github.com/hlxsites/servicenow/pull/69/commits/a0299367c1b62b6d764640341bcfa55002375af7